src/modding/ModManager.cs

```diff
@@ -408,6 +582,25 @@ public override void _Process(float delta)
         wasVisible = isCurrentlyVisible;
     }
 
+    public override void _Notification(int what)
+    {
+        if (what == NotificationTranslationChanged)
+        {
+            UpdateTabTitles();
+        }
+    }
+
+    public void UpdateLoadPosition(int startIndex = 0)
+    {
+        if (enabledMods != null)
+        {
+            for (int index = startIndex; index < enabledMods.Count; ++index)
+            {
+                enabledMods[index].LoadPosition = index;
+            }
+        }
+    }
```

Comment:
This would basically be the code I suggested adding to the mod loader, I think that method should be created in there and this updated to use that.